.github/workflows/pr-validations.yml

```diff
@@ -0,0 +1,27 @@
+# https://octokit.github.io/rest.js
+# https://github.com/actions/toolkit/blob/master/packages/github/src/context.ts
+
+name: PR Vadlidations
+on: pull_request
+
+jobs:
+  mandatory-changes:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
+      - name: Install packages
+        run: npm install --only=dev
+
+      - name: Validate
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        run: |
+          node ./.github/workflows/pr-validations.js mandatoryChanges
```

Comment:
Still not 100% sure why we are not implementing this as a proper node.js github action?

Reply:
Not really sure what you mean by that. There is no ready made action that implements the validations we want. So some code must be written. 

There are two options:

1. Create a GItHub action project --> publish it --> use it.
2. Use a github action that simply runs node.js code. Which is what i've done.

How did you imagine it would look like?

Reply:
I mean use [JavaScript Actions](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-actions#javascript-actions)

Reply:
More: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action

Reply:
Right so that is basically what I meant by option 1.

I just think it might be an overkill at this stage. I guess strictly speaking, what I implemented is a GitHub **Workflow**, not a GitHub **Action**.

My vision for this is as follows:

0. Create the validation code as a js runnable script.
1. Create a command line tool called `prlinter` (or something) that can be executed locally.
2. Create a github action that installs this tool and invokes it.
3. Publish this action
4. Use the action in `jsii` and `aws-cdk`

This PR implements stage 0, which already gives us nice value. I'll also probably implement step 1 as well actually, but i'm not sure we should be going any further for now. 

What we could do is create the action project locally, and reference it from the workflow:

```yaml
jobs:
  mandatory-changes:
     runs-on: ubuntu-latest
     steps:
 
       - name: Checkout
         uses: actions/checkout@v1

       - name: Validate Mandatory Changes
         uses: ./.github/actions/prlinter
         with:
           check: MANDATORY_CHANGES
```

This would be nice as it would make us better prepared for when we decide to publish it.
Is this what you meant?